### DISCUSSION TITLE
TurboRepo 2 application Next.js and Nest.js loading parallel

### DISCUSSION THREAD

**Discussion by willianrr** [vercel/community#1264] [Help] [answered]
Hello everyone, I'm putting together a monorepo project to run my frontend and backend using the stack nest.js and next.js However, I didn't find any tutorial or example on how to run these 2 applications using vercel. Does anyone have an idea how to do this?

**Comment by amyegan** [comment, marked as answer]
Hi @willianrr! You can find more information on deploying monorepo projects in the Vercel docs at https://vercel.com/docs/concepts/monorepos

The Next.js should be easy to deploy with the default Next.js preset configuration, but deploying Nest.js might be a little tricky. We don't have an official guide for it, but others have shared the `vercel.json` config they used to deploy a Nest.js project (i.e. https://stackoverflow.com/a/74644522)
